htdocs/js/notebook/notebook_controller.js

```diff
@@ -89,6 +89,12 @@ Notebook.create_controller = function(model)
             }
             model.user(notebook.user.login);
             model.update_files(notebook.files);
+
+            //
+            RCloud.UI.notebook_protection.userLogin = notebook.user.login;
+            RCloud.UI.notebook_protection.userId = notebook.user.id;
```

Comment:
you probably want to use editor.username() instead of this, which is the notebook user. it looks like (will test in a minute) if you load someone else's notebook, then the protection dialogs will be populated from that user instead of the logged-in user. also `userId` is unused.